packages/contractkit/src/wallets/signers/signer.ts

```diff
@@ -0,0 +1,25 @@
+// @ts-ignore-next-line
+import { account as Account, bytes as Bytes, hash as Hash, nat as Nat, RLP } from 'eth-lib'
+// @ts-ignore-next-line
+import * as helpers from 'web3-core-helpers'
+// @ts-ignore-next-line
+import { BN } from 'bn.js'
```

Comment:
What's the reason for all the `@ts-ignore-next-line`s here?
I'm curious if these are needed, since they don't appear to be referenced in this file. But, the fact that you added the ts-ignore suggests that they _are_ necessary in a way that's not understood by the linter.

I'd like to see a note next to those briefly explaining why, to help some future editor of the file understand what's safe to change/remove.

Reply:
Addressed in the previous comment.. thanks again!